CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: How to Contribute
+---
+Software Carpentry is an open source project,
+and we welcome contributions of all kinds:
+new lessons,
+fixes to existing material,
+bug reports,
+and reviews of proposed changes are all equally welcome.
+
+By contributing,
+you are agreeing that Software Carpentry may redistribute your work under
+[these licenses](LICENSE.md).
```

Comment:
On Thu, Feb 12, 2015 at 03:57:48AM -0800, Raniere Silva wrote:
> > +By contributing,
> > +you are agreeing that Software Carpentry may redistribute your work under
> > +[these licenses](LICENSE.md).
> 
> This link will be "broken" at `CONTRIBUTING.html`.

GitHub handles these links with it's file browser [1].  I don't know
if Pandoc does, but I don't think we want a CONTRIBUTING.html anyway.

[1]: https://github.com/blog/1395-relative-links-in-markup-files